Content/Fonts/retroLarge.spritefont

```diff
@@ -11,19 +11,19 @@ with.
     <!--
     Modify this string to change the font that will be imported.
     -->
-    <FontName>Verdana</FontName>
+    <FontName>Just My Type</FontName>

```

Comment:
the font file needs to be in the same directory as the `.spritefont` if you want the CI to pick it up. Since we can't install the font locally on the build machine.

Reply:
Ah ok. I'll make that change tomorrow.

Reply:
Stuff it. Who needs sleep. Done.